README.md

```diff
@@ -62,9 +79,54 @@ Alternatively, create a new config file named `config.json`,
             "commit": "6bd6b76d1156e20e45d1016f355d154141c7e5b9",
             "relative_root": "contracts"
         }
+    },
+    "raise_exception": true,
```

Comment:
What does 'raise_exception' mean in this context, appearing out of the blue?

Reply:
renamed to raise_exceptions_on_comparison

Reply:
1) It's still not clear — maybe it's better to describe the outcome, not the mechanism?
e.g.: `fail_on_comparison_error`
2) somewhere it's renamed, somewhere (like here) it's not :thinking: